ReportPortal.Tests/ReportPortal.E2E.API.Tests/Scenarios/DemoLaunchesTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using ReportPortal.E2E.API.Business;
+using ReportPortal.E2E.API.Business.Models.Responses;
+using ReportPortal.E2E.Core.Extensions;
+using ReportPortal.E2E.Core.Helpers;
+
+namespace ReportPortal.E2E.API.Tests.Scenarios
+{
+    public class DemoLaunchesTests : BaseNunitTest
+    {
+        private static readonly string ProjectName = RandomValuesHelper.RandomString();
+        private int _projectId;
+
+        protected override void Preconditions()
+        {
+            var createProjectResponse = Steps.AsAdminUser().CreateProject(ProjectName).GetAwaiter().GetResult();
+            createProjectResponse.EnsureSuccessStatusCode();
+            _projectId = createProjectResponse.GetResponse<CreateProjectResponse>().ProjectId;
+
+            var generateDataResponse = Steps.AsAdminUser().CreateDemoData(ProjectName).GetAwaiter().GetResult();
+            generateDataResponse.EnsureSuccessStatusCode();
+        }
+
+        //Dummy test
+        [Test]
+        public async Task Launch_Demo_Name()
+        {
+            var getLaunchesNamesResponse = await Steps.AsAdminUser().GetLaunchNames(ProjectName);
+            getLaunchesNamesResponse.EnsureSuccessStatusCode();
+            var names = getLaunchesNamesResponse.GetResponse<List<string>>();
+
+            names.Should().HaveCount(1);
+            names!.First().Should().Be("Demo Api Tests");
```

Comment:
Partially agree, because if you have 2 names, you can check that at least the first item is correct and you have some extra items if you use soft assertion. But if you keep current you can see that 0 or more that 1 names found and will see that first is not correct only on second iteration. This is a power of soft assertions that you can see all issues but only first.

Reply:
Left as Single without AssertionScope